bazel/test.bzl

```diff
@@ -9,7 +9,7 @@ running tests, like setting a reasonable number of cores and amount of memory.
 load("@bazel_skylib//rules:write_file.bzl", "write_file")
 load(":internal.bzl", "redpanda_copts")
 
-def has_flags(args, *flags):
+def _has_flags(args, *flags):
```

Comment:
is this a naming convention or something more?

Reply:
https://bazel.build/concepts/visibility#protecting-individual-symbols